exporter/util.py

```diff
@@ -101,8 +107,23 @@ def status(self) -> Literal["RUNNING", "COMPLETED", "WAITING"]:
             return "COMPLETED"
         return "WAITING"
 
-    def years_available(self) -> List[int]:
+    def files_available(self) -> Dict:
         """
-        Return the calendar years for which there are exported files in reverse chronological order.
+        Returns all the available file formats and segments (by year or full).
         """
-        return sorted((int(p.name[:4]) for p in self.directory.glob("[0-9][0-9][0-9][0-9].jsonl.gz")), reverse=True)
+        files = {}
+        # Ensure the template always receives expected keys.
+        for suffix in ("csv", "jsonl", "xlsx"):
+            files[suffix] = {"years": [], "full": False}
+
+        for path in self.directory.glob("*"):
+            suffix = path.name.split(".", 2)[1]
+            if suffix not in files:
+                continue
+            prefix = path.name[:4]  # year or "full"
```

Comment:
Currently, there are also JSON files by month, but this approach only takes the first four digits of the filename, which causes years duplication if there is more than one month per year, e.g.:
![image](https://user-images.githubusercontent.com/6834122/191550519-c81ef69c-3693-434d-91db-49da57171cad.png)
We can change this to also list files by month as per #215 or change this method to only return the JSONs by year and "full"

Reply:
Aha, I forgot. Hmm, job 713 has 210 files (2005-12 to 2022-06). Let's just do years for now and restore your deduplication. We can use a `set()`.